core/src/client/async_client/mod.rs

```diff
@@ -130,9 +138,33 @@ impl ClientBuilder {
 			id_manager: RequestIdManager::new(self.max_concurrent_requests, self.id_kind),
 		}
 	}
+
+	/// Build the client with given transport.
+	#[cfg(feature = "async-wasm-client")]
+	pub fn build_with_wasm<S, R>(self, sender: S, receiver: R) -> Client
+	where
+		S: TransportSenderT,
+		<S as TransportSenderT>::Error: Sync,
+		R: TransportReceiverT,
+		<R as TransportReceiverT>::Error: Sync,
+	{
+		let (to_back, from_front) = mpsc::channel(self.max_concurrent_requests);
+		let (err_tx, err_rx) = oneshot::channel();
+		let max_notifs_per_subscription = self.max_notifs_per_subscription;
+
+		wasm_bindgen_futures::spawn_local(async move {
```

Comment:
https://crates.io/crates/wasm-futures-executor is a candidate but doesn't compile for stable rust yet....

Reply:
I guess we could call the WASM client "experimental" and require nightly until the `wasm-futures-executor` stabilizes?

Reply:
it seems be fast enough for now, let's open a issue and investigate the performance gain for it instead....